client/app/dbaas/logs/detail/options/logs-options.service.js

```diff
@@ -15,7 +15,7 @@ class LogsOptionsService {
      * @returns the transformed option
      * @memberof LogsOptionsService
      */
-    transformOption (option) {
+    _transformOption (option) {
```

Comment:
yay _ private notation

Reply:
It is to indicate that it is a private function. Do we follow that in OVH manager? should I remove?

Reply:
Yup I do that myself.  I think it's clearer this way ^^